extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -225,17 +210,6 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 			if ( ( $action == "match" || $action == "check" ) ) {
 				if ( !is_null( $this->user ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)